File: src/main/java/project/rpg/listeners/PlayerQuitEventListener.java
package project.rpg.listeners;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerQuitEvent;
import project.rpg.ui.ActionBarUI;

public class PlayerQuitEventListener implements Listener {

    @EventHandler
    public void onPlayerQuit(PlayerQuitEvent event){
        ActionBarUI.deletePlayer(event.getPlayer());
    }

}

File: src/main/java/project/rpg/listeners/PlayerQuitEventListener.kt
package project.rpg.listeners;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerQuitEvent;
import project.rpg.player.PlayerInformation;

public class PlayerQuitEventListener implements Listener {

    @EventHandler
    public void onPlayerQuit(PlayerQuitEvent event){
        PlayerInformation.deleteInfo(event.getPlayer());
    }

}
